src/class/mms-project.mjs

```diff
@@ -351,6 +407,20 @@ export class MmsProject {
 		return await upload(ds_upload, this._endpoint_ref('elements', si_ref, {overwrite:true}), this._gc_req_post);
 	}
 
+	async upload_passthrough(elements, si_ref, option) {
+		const ds_passthrough = new stream.PassThrough();
+		const dp_upload = upload(ds_passthrough, this._endpoint_ref('elements', si_ref, {overwrite:true}), option);
+
+		ds_passthrough.write(/* syntax: json */ `{"elements":[`);
+		let i_element = 0;
+		for(const si_element of elements) {
+			ds_passthrough.write((i_element++? ',': '')+/* syntax: json */ `\n{"id":"${si_element}"}`);
```

Comment:
for example, this line only writes the id of each element which is ok for deletions, but not additions